Information/GetGlobalPlacement.FCMacro

```diff
@@ -113,13 +87,9 @@ def get_global_placement(root_object: app.DocumentObject,
             primitve" in PartDesign.
         - 'Body.Box001.Face6' if you select the top face of a Part->Body->
             "additive primitve" in PartDesign.
-
     """
-    objects = get_subobjects_by_full_name(root_object, subobject_fullpath)
-    placement = root_object.Placement
-    for o in objects:
-        placement *= o.Placement
-    return placement
+
+    return get_global_placement_and_scale(object, subobject_fullpath)[0]
```

Comment:
The case where `get_global_placement_and_scale()` returns `None` is not handled.